src/frontend/web_application/src/components/Icon/index.jsx

```diff
@@ -76,7 +78,7 @@ Icon.propTypes = {
 };
 Icon.defaultProps = {
   className: undefined,
-  type: undefined,
+  type: false,
```

Comment:
Ok sorry this comes from a very bad file manipulation from me. I didn't edit this, I copied the file from an old branch before: https://github.com/CaliOpen/Caliopen/commit/cf1c15f3423a4317e2a48b28ad40dda8d00d0f5e 
when I didn't know how to revert a commit (... I know it's not good)

btw of course I understand `undefined` match perfectly this prop :)